Automattic/WP-Job-Manager
# Add psalm

### Changes Proposed in this Pull Request

* Add Psalm static analysis tool

### Testing Instructions

* Check if the action for psalm are running fine
* Run `composer install` 
* Play with the code, using `mixed` on the code as return type, etc.
* Run `./vendor/bin/psalm --php-version=7.4` to check if it catches errors on PHP 7.4
* Run `./vendor/bin/psalm --php-version=8.2` to check if it catches errors on PHP 8.2


### Release Notes

### Discussion

The related post for discussion is p6r3EZ-21n-p2 





----

| Plugin build for 7aa51c9aa80776576d13c7ef76d407adacf33a5a <a href="#"><img width=600></a> |
| ------------------------------------------------------------ |
| 📦 [Download plugin zip](https://wpjobmanager.com/wp-content/uploads/2024/02/wp-job-manager-zip-2735-7aa51c9a.zip)                       |
| ▶️ [Open in playground](https://wpjobmanager.com/playground/?core=2024/02/2735-7aa51c9a)             |
